pkg/fork-tests/tests/silo-linear-pool-v2/readme.md

```diff
@@ -1,9 +1,11 @@
-# Silo Linear Pool
+# Silo Linear Pool v2
 
-First deployment of the `SiloLinearPoolFactory`, for Linear Pools with a Silo yield-bearing token (shareToken).
+Second deployment of the `SiloLinearPoolFactory`, for Linear Pools with a Silo yield-bearing token (shareToken).
 Already fixes the reentrancy issue described in https://forum.balancer.fi/t/reentrancy-vulnerability-scope-expanded/4345.
 Also has a fix in the `SiloLinearPoolRebalancer` to handle tokens which require the `SafeERC20` library for approvals.
 
+Supersedes `silo-linear-pool`, updating the BasePoolFactory `_create` function, by requiring an additional `uint256 salt` parameter.
```

Comment:
Add the date: https://github.com/balancer/balancer-v2-monorepo/tree/master/pkg/deployments/tasks/20230315-silo-linear-pool